v2.0/app/containers/MassEnergizeSuperAdmin/_FormGenerator/icon_files.json

```diff
@@ -0,0 +1,790 @@
+{
+    "ICON_FILES": [
```

Comment:
Also what is happening here? I am actually curious. Is there a reason we need such a long list and such a large file? 
Are trying to implement fontawesome icons here...? 
Why are we not using the npm package instead?